# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- thunderbird-128.0.1/toolkit/mozapps/update/updater/updater.cpp.vanilla	2024-08-01 22:15:05.881448120 +0200
+++ thunderbird-128.0.1/toolkit/mozapps/update/updater/updater.cpp	2024-08-01 22:15:34.312448209 +0200
@@ -107,9 +107,9 @@
 #  define USE_EXECV
 #endif
 
-#if defined(XP_OPENBSD)
+//#if defined(XP_OPENBSD)
 #  define stat64 stat
-#endif
+//#endif
 
 #if defined(MOZ_VERIFY_MAR_SIGNATURE) && defined(MAR_NSS)
 #  include "nss.h"
